build/src/main/resources/functions/RefreshLocationThresholdsData/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "name": "message",
+      "type": "serviceBusTrigger",
+      "direction": "in",
+      "queueName": "fews-location-thresholds-queue",
```

Comment:
This name was a bit of a guess and will need to be confirmed. I did ask for this information early on in development but it wasn't something we had at the time.